ymer/src/main/java/com/avanza/ymer/MongoDocumentCollection.java

```diff
@@ -158,6 +162,12 @@ public void updatePartialByIds(Set<Object> ids, Map<String, Object> fieldsToSet)
 					writeModels.add(updateManyModel);
 				}
 			}
+
+			@Override
+			public void unsetFieldsPartialByIds(Set<Object> ids, Set<String> fieldsToUnset) {
+				Bson updates = toFieldDeletes(fieldsToUnset);
+				addUpdates(ids, updates);
+			}
```

Comment:
We can remove `@SuppressWarnings("Convert2Lambda")` on the `bulkWrite` now. I put it there so the IDE would not suggest converting it to a lambda, since it was not meant to be a function interface, just happened to have a single method initially. It was meant to be expanded with more methods, just like you did 👍

Reply:
👍